src/org/labkey/test/util/core/webdav/WebDavUploadHelper.java

```diff
@@ -133,6 +133,36 @@ public void uploadFile(@NotNull File file)
         uploadFile(file, "");
     }
 
+    public void deleteFile(@NotNull String file, boolean shouldFail)
+    {
+        try
+        {
+            _sardine.delete(file);
```

Comment:
This should use the UrlFactory so that callers don't have to.
```suggestion
            _sardine.delete(_urlFactory.getPath(file));
```